book/src/user/tracing.md

```diff
@@ -12,10 +12,15 @@ if the config had `endpoint_addr = '127.0.0.1:3000'`, then
 
 See the [`filter`][filter] documentation for more details.
 
-Zebra also has support for generating [flamegraphs] of tracing spans,
-configured using the [`flamegraph`][flamegraph] option.
+Zebra also have support for:
+
+* Generating [flamegraphs] of tracing spans, configured using the
+[`flamegraph`][flamegraph] option.
+* Sending tracing spans and events to [systemd-journald][systemd_journald],
+on Linux distributions that use `systemd`.
```

Comment:
Do you think we should mention the use_journald option here?

I don't really have a strong opinion either way.